### PR TITLE
doc: enhance readme.

### PR DESCRIPTION
**Pull request by ldez**
Clarify readme and options spotlight.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,39 @@
 
 Shows the rendered HTML of the AsciiDoc content in current editor using <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd>.
 
-It can be activated from the editor using the <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd> key-binding and is currently enabled for `.adoc`, `.asciidoc`, `.ad`, `.asc`, `.adoc.txt`, `.txt` files.
+It is currently enabled for `.adoc`, `.asciidoc`, `.ad`, `.asc`, `.adoc.txt`, `.txt` files.
 
-**Note:** since 1.0.0, the autocompletions have been migrated to a new package: [autocomplete-asciidoc](https://atom.io/packages/autocomplete-asciidoc)
+You can toggle the preview pane, which appears to the right of the editor, using the key combination <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd>.
```
Comment (Mogztter) [inline README.md]: This sentence is redundant with the previous one no ? What about:

"Preview AsciiDoc content as HTML for `.adoc`, `.asciidoc`, `.ad`, `.asc`, `.adoc.txt`, `.txt` files."
"You can toggle the preview pane, which appears to the right of the editor, using the key combination <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd>."

Comment (ldez) [inline README.md]: I think "You can toggle the preview pane ..." can be simply suppressed

Comment (Mogztter) [inline README.md]: Why not but in my opinion this sentence helps to understand the screenshot.
I think the first sentence shouldn't mentioned shortcuts but just what this package is doing (not how)

Comment (ldez) [inline README.md]: I have remove shortcuts from the first sentence and move "You can toggle..." below the first sentence.

**Pull request merged**